x-pack/plugins/security_solution/public/timelines/store/middlewares/timeline_pinned_event.ts

```diff
@@ -64,17 +64,10 @@ export const addPinnedEventToTimelineMiddleware: (kibana: CoreStart) => Middlewa
           timelineId: timeline.savedObjectId,
         });
 
-        const response = result.data.persistPinnedEventOnTimeline;
-        if (response && 'code' in response && response.code === 403) {
-          store.dispatch(showCallOutUnauthorizedMsg());
-        }
-
         refreshTimelines(store.getState());
 
         const currentTimeline = selectTimelineById(store.getState(), action.payload.id);
-        // The response is null or empty in case we unpinned an event.
-        // In that case we want to remove the locally pinned event.
-        if (!response || !('eventId' in response)) {
+        if ('unpinned' in response) {
```

Comment:
'aha' 👍🏾 the `unpinned` makes sense to me now. Not necessary for this, but this makes me feel even more so that we should just have an `unpinPinnedEvent` api that's essentially a `delete`. Thanks for making this compromise on short notice